profile.go

```diff
@@ -159,6 +163,31 @@ fetching may be degraded.
 			return nil
 		},
 	},
+	"randomports": {
+		Description: `Use a random port number for swarm.`,
```

Comment:
Hello @djdv  
as you can see here I updated the discretion as you said.

again sorry be late and please feel free to ask about more update.